apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-Schl\u00fcssel",

```

Comment:
@jforscht all files should be written in English language. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings). Please provide a proper displayName, desctioption, and tooltip in English language.